### ISSUE TITLE
Tee output to stdout?

### ISSUE THREAD

**Issue by j2kun** [jupyter/nbconvert#374]
Hey nbconvert folks, I've got another question:

I've been using nbconvert's execute preprocessor to run some jupyter notebooks from the command line, and I've been wondering if there's a way to `tee` the cell output to stdout as the notebook runs. Is this currently possible with nbconvert, and if not would it be feasible to add this feature? (I'd be happy to submit a PR)

**Comment by takluyver** [comment]
No, I don't think it's currently possible, but I think it would be a feasible thing to add. Look out for #360, which also changes the relevant code now - if you try to fix this in parallel, the PRs will clash.

**Comment by RoyalTS** [comment]
This would be great to have. Has there been any progress on this? Is it even on anyone's agenda?

**Comment by RoyalTS** [comment]
Not a native solution but this does what you want it to do: https://github.com/guoquan/runnb

**Comment by InnovArul** [comment]
Just an addition to @RoyalTS's comment. You can install runnb using pip: 
`pip install runnb`
https://pypi.org/project/runnb

**Comment by MSeal** [comment]
Also https://papermill.readthedocs.io/en/latest/ has an option to tee outputs, though papermill only does notebook-to-notebook execution (it wraps nbconvert)

**Comment by RoyalTS** [comment]
Yeah, I've settled on papermill for this.

**Comment by matthewwardrop** [comment]
For anyone looking for not only tee'd output, but also access to the variables created by the notebook, I've created an in-process runner here: https://gist.github.com/matthewwardrop/fe2148923048baabe14edacb2eda0b74 .

**Comment by Hoeze** [comment]
Is this feature available yet?
Would be useful to track progress in e.g. snakemake.

**Comment by MSeal** [comment]
@Hoeze  I believe general consensus is still to use papermill for output teeing rather than nbconvert.

**Comment by VictorDataEngineer** [comment]
Hi, 
  
   I am also looking to stdout only the notebook cell output (text) by executing the papermilll command in terminal, but I get a lot of other information along it.

Here is the command:
`papermill test.ipynb -`

The notebook contains simple print('Hello World') , and in the stdout I expect to see only: Hello World as plain text.

Also tried to set other configurations, as:   --report-mode / --no-report-mode; log-level; log-output...

Does this work in the same way on your side ?

**Comment by alonme** [comment]
@MSeal - How is this achieved with papermill?
